src/stores/AccountsStore.js

```diff
@@ -220,4 +234,132 @@ export default class AccountsStore extends Container<AccountsState> {
 	getAccounts() {
 		return this.state.accounts;
 	}
+
+	getLegacySubstrateAccounts() {
+		const result = [];
+		const accounts = this.state.accounts[Symbol.iterator]();
+		for (let [key, value] of accounts) {
+			if (key.split(':')[0] === NetworkProtocols.SUBSTRATE) {
+				result.push([key, value]);
+			}
+		}
+		return result;
+	}
+
+	async unlockIdentitySeed(pin) {
+		const { encryptedSeed } = this.state.currentIdentity;
+		const seed = await decryptData(encryptedSeed, pin);
+		const { phrase } = parseSURI(seed);
+		return phrase;
+	}
+
+	getNewIdentity() {
+		return this.state.newIdentity;
+	}
+
+	async saveNewIdentity(seedPhrase, pin) {
+		const updatedIdentity = deepCopyIdentity(this.state.newIdentity);
+		updatedIdentity.encryptedSeed = await encryptData(seedPhrase, pin);
+		const newIdentities = this.state.identities.concat(updatedIdentity);
+		this.setState({
+			currentIdentity: updatedIdentity,
+			identities: newIdentities,
+			newIdentity: emptyIdentity()
+		});
+		await saveIdentities(newIdentities);
+	}
+
+	async selectIdentity(identity) {
+		await this.setState({ currentIdentity: identity });
+	}
+
+	updateNewIdentity(identityUpdate) {
+		this.setState({
+			newIdentity: { ...this.state.newIdentity, ...identityUpdate }
+		});
+	}
+
+	async updateCurrentToIdentities() {
```

Comment:
To enforce better developer experience for all of us and for the future contributors, let's all aim to be more thorough with inline comments and JSDoc. 

This function name is not particularly clear at a glance what it does. Could you please add a/some comments and maybe make the function name more specific?

Reply:
Agree, actually I prefer to use explicit function name with minimal comments.